mcan/src/bus.rs

```diff
@@ -157,13 +157,26 @@ pub trait DynAux {
     /// disable CAN operation.
     fn initialization_mode(&self);
 
-    /// Sets MCAN up for being shut down.
+    /// Requests the peripheral to enter "power down" mode.
+    ///
     /// See `Power Down (sleep mode)`, in the peripheral docs, for user
     /// consideration regarding clocking and message handling.
-    fn shutdown(&self);
+    ///
+    /// It is also worth noting that this mode should not be interpreted as the
+    /// peripheral being "powered off", since it is still possible to
+    /// configure the peripheral while in this mode.
+    fn power_down_mode(&self);
 
-    /// Check if csa bit is set
-    fn ready_for_shutdown(&self) -> bool;
+    /// Check if the transition to `Power Down (sleep mode)` is complete and
+    /// that it is safe to completely disable the peripheral.
+    ///
+    /// Returns `false` if the peripheral is still handling outgoing or
+    /// incomming messages.
+    ///
+    /// If the bus is heavily congested, the peripheral
+    /// might never enter `Power Down (sleep mode)` on its own. In that case
+    /// it can be forced by calling `initialization_mode`.
+    fn is_ready_for_power_off(&self) -> bool;
```

Comment:
Nit: Was "off" intentionally chosen and not "down"?

Reply:
I mentioned it above:
https://github.com/GrepitAB/mcan/pull/45#discussion_r1500636372

> I renamed it to is_ready_for_power_off to make the difference between "power down mode" and "power off" even clearer.

>What do you think?

Reply:
Sorry, I missed that. Looks good!